metricbeat-yml-script.py

```diff
@@ -180,17 +181,20 @@ def _add_aws_shipping_data():
 
 
 def _get_tags_value(aws_namespace):
-    service_name = aws_namespace.split("/")[1].lower()
-    if service_name == "ebs":
-        return "ec2:snapshot"
-    elif service_name == "elb" or service_name == "applicationelb" or service_name == "networkelb":
-        return "elasticloadbalancing"
-    elif service_name == "amazonmq":
-        return "mq"
-    elif service_name == "efs":
-        return "elasticfilesystem"
+    if "aws/" in aws_namespace.lower():
+        service_name = aws_namespace.split("/")[1].lower()
+        if service_name == "ebs":
+            return "ec2:snapshot"
+        elif service_name == "elb" or service_name == "applicationelb" or service_name == "networkelb":
+            return "elasticloadbalancing"
+        elif service_name == "amazonmq":
+            return "mq"
+        elif service_name == "efs":
+            return "elasticfilesystem"
+        else:
+            return service_name
     else:
-        return service_name
+        return aws_namespace.lower()
```

Comment:
@mirii1994 what will happen to metricbeat if I'll enter a namespace that doesn't exists?

Reply:
From what I've seen, there is no message from metricbeat that indicated whether the namespace you entered exists. If you enter it with other existing namespaces it won't affect them - the existing namespaces' metrics will arrive. If you enter only that non-existing namespace then it looks like metricbeat is running but nothing you won't get any metrics.